pkg/appruntime/agent/executor.go

```diff
@@ -71,6 +71,9 @@ func (p *Executor) Run(ctx context.Context, cli client.Client, args map[string]a
 				continue
 			}
 			allowedTools = append(allowedTools, tool)
+		case "calculator":
```

Comment:
we can later add one agent test in tests/example-test.sh for github action tests for each pull request.

Reply:
yes, let's do it later